processor/probabilisticsamplerprocessor/factory.go

```diff
@@ -44,7 +44,8 @@ func NewFactory() component.ProcessorFactory {
 	return component.NewProcessorFactory(
 		typeStr,
 		createDefaultConfig,
-		component.WithTracesProcessor(createTracesProcessor, stability))
+		component.WithTracesProcessor(createTracesProcessor, stability),
+		component.WithLogsProcessor(createLogsProcessor, stability))
```

Comment:
The logs processor should be Alpha, given there are no known production users of this feature at this point.